firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/BreathingCircuit/AlarmMuteService.h

```diff
@@ -14,11 +14,23 @@
 
 namespace Pufferfish::Driver::BreathingCircuit {
 
-static const float countdown_time = 120;
+static const uint64_t countdown_time = 120;
 
 class AlarmMuteService {
  public:
-  static void transform(AlarmMute &alarm_mute, const AlarmMuteRequest &alarm_mute_request);
+  void transform(
+      uint32_t current_time, const AlarmMuteRequest &alarm_mute_request, AlarmMute &alarm_mute);
+
+ private:
+  void continue_countdown(AlarmMute &alarm_mute);
+  static const uint32_t sensor_update_interval = 1000;  // ms
```

Comment:
Let's try to remove `sensor_update_interval`, `update_needed`, and `previous_time_` if possible.

Reply:
Yep removed!